pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -508,6 +489,29 @@ func (c *component) createOrPatch(ctx context.Context, sts *appsv1.StatefulSet,
 		if c.values.PriorityClassName != nil {
 			sts.Spec.Template.Spec.PriorityClassName = *c.values.PriorityClassName
 		}
+		if c.values.UseEtcdWrapper {
+			// sections to add only when using etcd wrapper
+			// TODO: @aaronfern add this back to sts.Spec when UseEtcdWrapper becomes GA
+			sts.Spec.Template.Spec.InitContainers = []corev1.Container{
+				{
+					Name:         "change-permissions",
+					Image:        "alpine:3.18.2",
+					Command:      []string{"sh", "-c", "--"},
+					Args:         []string{"chown -R 65532:65532 /var/etcd/data"},
```

Comment:
Shouldn't the args be in separate elements within the string array?
```suggestion
					Args:         []string{"chown", "-R", "65532:65532", "/var/etcd/data"},
```

Reply:
Not really, it shouldn't make a difference. However the suggestion reads better, so I've added it